### ISSUE TITLE
Cannot boot Windows 11

### ISSUE THREAD

**Issue by wwwwjfjf** [axemre/OpenCore_HP_ProBook440G5_Ventura#1]
I'm using this EFI but I cannot boot Windows! This should be a Bug……?

**Comment by axemre** [comment]
Hi. I have tried booting Windows 11 and it seems to be working perfectly fine. I'll update it anyways. Make sure that you've set correct BIOS settings and the boot order. Are you able to boot Windows normally? (Without OpenCore)

Let me know.

**Comment by wwwwjfjf** [comment]
Yep. I used the another edition (yours but not Ventura) and it seemed ok.
And could you send me a installation dmg file with onedrive? Thank you!
(PS: I'm a Chinese so I could not use google, you know.)

**Comment by wwwwjfjf** [comment]
It stoped when the circle went a half
like this:
![无标题](https://user-images.githubusercontent.com/68053997/211176549-3307ca9e-27a2-44e2-aa8b-b9a41c8f4f9d.png)

**Comment by axemre** [comment]
Ah, I see. Gonna update that later. This happens because of OC. And even if Windows boots, OC will inject ACPI into WIndows, and that'll break brightness keys and change system board name to Acidanthera-Mac. I'm gonna check that, right now I just don't have the time to do it.

**Comment by wwwwjfjf** [comment]
Eh.....what??? Sorry I could not understand (new of hackintosh) .
I just used github to search my HP Probook.

**Comment by wwwwjfjf** [comment]
And I'm wondering that I could not use that OC for Monterey to update to Ventura.
Does OC (version 0.8.0) support it? Pls let me know.
